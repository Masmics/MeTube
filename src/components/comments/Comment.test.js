import React from 'react';
import { shallow } from 'enzyme';
import Comment from './Comment';

describe('comment component', () => {
  it('renders a video comment', () => {
    const wrapper = shallow(<Comment comment="Sweet video!" />);
    expect(wrapper).toMatchSnapshot();
  });
});
