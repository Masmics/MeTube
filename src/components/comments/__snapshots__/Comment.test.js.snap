// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`comment component renders a video comment 1`] = `
<p>
  Sweet video!
</p>
`;
